### ISSUE TITLE
Improve accessibility of color picker used in book info page

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#2437]
> The contents of the color picker do not have labels (color swatches, focusable elements). When tabbing into the color picker the different elements are communicated as “clickable”,  or “group” (other screen readers may communicate the elements using other generic, non-descript terms) but no feedback is given about their behavior or the values the user is selecting. While making such an interaction fully accessible to screen reader users may be challenging, labelling some of the elements could make the interaction a lot more clear, so that users at least know the functionality of the elements they are on. For example, labelling the color selector and standard swatches (black, white, red, orange, etc.). Alternatively, since users are also able to modify the color by typing a hex code into a text field, the color selector could be hidden using aria-hidden so that keyboard users are still able to interact with the elements, but screen reader and braille display users are not confused by interacting with 10+ unlabeled elements which are all announced simply as “clickable”. Note that labelling elements is recommended over hiding them since this can be beneficial for partially sighted users who still rely on a screen reader. Ultimately with an interaction this complex, user research is required to design an ideal interaction that supports a wide range of users.

**Comment by SteelWagstaff** [comment]
Resolved by https://github.com/pressbooks/pressbooks/pull/4049
